src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/app.css

```diff
@@ -22,6 +22,28 @@ a, .btn-link {
     padding-top: 1.1rem;
 }
 
+svg {
+    width: 200px;
+    height: 200px;
```

Comment:
Should we possibly be using [relative units](https://www.w3schools.com/CSSref/css_units.asp) here? Have you had a chance to test out the responsiveness of the animation (ie. mobile/tablet/high res screens, etc.)

Reply:
I can change these to relative units, but the SVG and animation seem to scale when I test using the device emulation on developer tools.